src/app/Services/ImageTransformer.php

```diff
@@ -4,13 +4,13 @@
 
 use Illuminate\Http\UploadedFile;
 use Intervention\Image\Facades\Image;
-use LaravelEnso\ImageTransformer\app\Exceptions\ImageTransformerException;
-use LaravelEnso\ImageTransformer\app\Exceptions\MissingDependencyException;
+use LaravelEnso\ImageTransformer\app\Exceptions\ImageTransformer as ImageTransformerException;
```

Comment:
we don't need aliases here

Reply:
we import 2 exceptions here and one the exceptions has the same name as the class itself. How should I refactor?